### PR TITLE
feat: Disable modal on catalog page

### LINKED ISSUE #1232: [Enhancement] Disable the modal on catalog page
### Current Behavior
1. Clicking on any card on [catalog](https://meshery.io/catalog) page a modal pops up on the screen
2. On clicking the redirect icon on modal -> opens a new tab with catalog info

Group 1116599533 (1)

### Desired Behavior
1. Disable the modal shown above
2. Clicking on any card on [catalog](https://meshery.io/catalog) page should open the catalog info page in the same tab instead of a new tab

### Screenshots / Mockups

---
<h4>Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources</h4>

- 🎨 Wireframes and designs for Meshery Site in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Ghat0tkach**
Commented the card-model out rather than deleting for future references

**Description**

This PR fixes #1232 
Latest [Preview](https://deploy-preview-1241--mesheryio-preview.netlify.app/)
**Notes for Reviewers**
Commented out the card-model code 

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Ghat0tkach** [comment]
https://layer5io.slack.com/archives/C023QGMSH8D/p1688143670586869

**Comment by vishalvivekm** [comment]
@Ghat0tkach Let's discuss this on the Websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#) if you would. :)

**Comment by Ghat0tkach** [comment]
> @Ghat0tkach Let's discuss this on the Websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#) if you would. :)

I have been having some issues with my system right now ,if it get resolved within today . I'll be there🙌

**Comment by leecalcote** [comment]
Thanks for this, @Ghat0tkach

File: _includes/card.html [hunk 1]
```diff
@@ -1,8 +1,10 @@
+<a  href="{{pattern.url}}">
   <div class= "card link">  
     <div class="card-inner">
-
+
```
Comment (thisiskaransgit) [inline _includes/card.html:4]:
```suggestion

```

File: _includes/card.html [hunk 2]
```diff
@@ -1,8 +1,10 @@
+<a  href="{{pattern.url}}">
   <div class= "card link">  
     <div class="card-inner">
-
+    
       <div class="front">
         <div class="chip">
+
```
Comment (thisiskaransgit) [inline _includes/card.html:7]:
```suggestion

```

**Review by thisiskaransgit — APPROVED** [review]
Looks good to merge, thank you for your contribution :tada:

**Comment by Ghat0tkach** [comment]
Thank you @thisiskaransgit , done the suggested changes 🙌

**Comment by samyakshah3008** [comment]
@Ghat0tkach  Let's discuss this on Websites Meeting call on Monday. Please add this as an agenda item in the [meeting mintes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) if you would. :)

**Comment by Ghat0tkach** [comment]
> @Ghat0tkach  Let's discuss this on Websites Meeting call on Monday. Please add this as an agenda item in the [meeting mintes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) if you would. :)

@samyakshah3008 thanks a lot however we discussed this in previous call 😅.

**Comment by samyakshah3008** [comment]
> > @Ghat0tkach  Let's discuss this on Websites Meeting call on Monday. Please add this as an agenda item in the [meeting mintes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) if you would. :)
> 
> @samyakshah3008 thanks a lot however we discussed this in previous call 😅.

Sounds cool!

**Pull request merged**